basics/a29.go

```diff
@@ -0,0 +1,21 @@
+// alias declarations
+
+package main
+
+import "fmt"
+
+func main() {
+	type A = string // defining a new data type A w
+	var a A
+	a = "foo"
+	a += "bar"
+	fmt.Println(a)
+	fmt.Printf("Type of a is %T", a) // this prints string
+	type B string
+	var b B
+	b = "foo"
+	b += "baz"
+	fmt.Println(b)
+	fmt.Printf("Type of b is %T", b) // this should print B
```

Comment:
Similarly I see "Type of b is main.B% "

Reply:
i am not sure, need to research but in this case B is new type and not an alias like A.